src/test/resources/org/openstreetmap/atlas/geography/geojson/parser/featureChangePropertiesExample2.json

```diff
@@ -41,7 +41,7 @@
           "name": "RELATION_MEMBER",
           "type": "ADD",
           "itemType": "NODE",
-          "id": 402306209000000,
+          "id": -1374305525422343588,
```

Comment:
I am curious as to why this test ID had to change?

Reply:
This was actually just to test that the parser correctly deserialized this id since it wasn't doing so before the changes. I just picked a feature from existing test data and assigned it a new id to test this out. Only usage of this file appears to be for this test so I wasn't concerned with keeping ids consistent with OSM